app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command_line: str) -> None:
+    command, old_file, new_file = command_line.split()
+
+    if command == "cp" and old_file != new_file:
+        with open(old_file, "r") as file_in, open(new_file, "w") as file_out:
+            file_out.write(file_in.read())
+            print(f"File {old_file} has been copied to the file {new_file}")
```

Comment:
this is a requirement from checklist to the task, fix this please
![image](https://user-images.githubusercontent.com/103201010/219686320-5acded9b-a8c8-4778-92f2-98185420f0b4.png)

Reply:
Done